db/schema.rb

```diff
@@ -306,7 +324,6 @@
     t.index ["uid"], name: "index_users_on_uid"
   end
 
-  add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
```

Comment:
Do we know what this was?

Reply:
I added that back. When I recreated the schema this did not show up so that is worth looking it.